bedrock/firefox/templates/firefox/releases/release-notes.html

```diff
@@ -129,11 +129,19 @@ <h1>{{ _('See what’s new in Firefox!') }}</h1>
           <div class="version">
             <h2>{{ _('{version}')|f(version=release.version) }}</h2>
             <h3>{{ release.product }} {{ release.channel }}</h3>
-            <p>{{ _('{date}')|f(date=release.release_date|l10n_format_date) }}</p>
+            {% if release.is_public or settings.DEV %}
```

Comment:
Instead of having to do `or settings.DEV` everywhere, let's put a new property method on the ProductRelease model that returns `self.is_public or settings.DEV`. Should keep the templates a bit more clean. In fact, we already read in the individual notes as `is_public = True` when the site is in DEV mode when the notes are imported, perhaps we should set all notes in the DB as `is_public = True` on import as well when DEV is true and just not have to worry with it at run time?